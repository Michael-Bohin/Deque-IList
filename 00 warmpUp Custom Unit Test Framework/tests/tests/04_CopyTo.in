new int
add 4 
add 290367140
add 228125465
add 349822234
add 847257380
add 705661013
add 753158746
add 212333020
add 889655868
add 366759348
add 852761187
count 
foreach
copyto
clear
add 30
count
foreach

new string 
add ahoj
add svete
add unit
add testing
count
foreach
copyto
clear
copyto
add xxxxxxx
copyto
add yy
copyto
add zz
copyto
add a
add b
copyto
add c
add d
add e
add f
copyto