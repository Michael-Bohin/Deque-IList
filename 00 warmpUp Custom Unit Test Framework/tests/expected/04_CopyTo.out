11
4
290367140
228125465
349822234
847257380
705661013
753158746
212333020
889655868
366759348
852761187
1,2,3,4,5,4,290367140,228125465,349822234,847257380,705661013,753158746,212333020,889655868,366759348,852761187,
1
30
4
ahoj
svete
unit
testing
ykRXzPdlOZ,ahoj,svete,unit,testing,flQeR,rxFhGKVS,YyvLhLZw,FCZMrDSFIg,OpvvGxAdij,
ykRXzPdlOZ,FquDbpgHyX,ZttqHzmFtB,RvNzjcrLtK,WxhwlaFwOy,flQeR,rxFhGKVS,YyvLhLZw,FCZMrDSFIg,OpvvGxAdij,
ykRXzPdlOZ,xxxxxxx,ZttqHzmFtB,RvNzjcrLtK,WxhwlaFwOy,flQeR,rxFhGKVS,YyvLhLZw,FCZMrDSFIg,OpvvGxAdij,
ykRXzPdlOZ,xxxxxxx,yy,RvNzjcrLtK,WxhwlaFwOy,flQeR,rxFhGKVS,YyvLhLZw,FCZMrDSFIg,OpvvGxAdij,
ykRXzPdlOZ,xxxxxxx,yy,zz,WxhwlaFwOy,flQeR,rxFhGKVS,YyvLhLZw,FCZMrDSFIg,OpvvGxAdij,
ykRXzPdlOZ,xxxxxxx,yy,zz,a,b,rxFhGKVS,YyvLhLZw,FCZMrDSFIg,OpvvGxAdij,
ykRXzPdlOZ,xxxxxxx,yy,zz,a,b,c,d,e,f,
